Makefile

```diff
@@ -80,6 +80,12 @@ else
   colorize = $(shell printf >&2 "\033[33m$1\033[0m\n")
 endif
 
+ifneq ($(LLVM_VERSION),)
+  ifeq ($(shell test $(firstword $(subst ., ,$(LLVM_VERSION))) -ge 18; echo $$?),0)
+    DEPS =
+  endif
+endif
```

Comment:
**suggestion:** Could we move this visually closer to the definition of `DEPS` to make it easier to reason about?
Or even put everything into a single conditional?
```
if (LLVM < 18)
  DEPS = $(LLVM_EXT_OBJ)
else
  DEPS =
endif
```